core/state_processor.go

```diff
@@ -155,3 +155,10 @@ func ApplyTransaction(config *params.ChainConfig, bc vm.ChainContext, txFeeRecip
 
 	return receipt, err
 }
+
+// StateProcessResult represents processing results from StateProcessor.
+type StateProcessResult struct {
```

Comment:
Even if it makes sense here from a semantics perspecitve. It's not really used here. 

I would move this type inside instabul modules, to the same place where we define the interface for `Verify()` and `Commit()` methods

Reply:
Put it in `consensus/istanbul/core/roundstate.go`, that's where it's used in the lowest level.